Userland/Utilities/js.cpp

```diff
@@ -317,11 +317,11 @@ static void print_object(JS::Object& object, HashTable<JS::Object*>& seen_object
     js_out("}}");
 }
 
```

Comment:
Since I assume #14437 will be merged first, this is a reminder to update it in the first commit as well.